server/services/taskService.test.ts

```diff
@@ -103,6 +104,50 @@ describe('taskService', () => {
       expect(taskClientFactory).toHaveBeenCalledWith(token)
       expect(taskClient.find).toHaveBeenCalledWith(applicationId, 'assessment')
     })
+
+    it('filters users result by AP area and qualification', async () => {
+      const apArea = {
+        id: '0544d95a-f6bb-43f8-9be7-aae66e3bf244',
+        name: 'Midlands',
+      }
+
+      const qualification = 'womens' as UserQualification
```

Comment:
`const qualification: UserQualification = 'womens' as const` 
      is preferable as it retains type safety